README.md

```diff
@@ -155,10 +293,31 @@ using LoggingExtras
 using Logging
 using HTTP
 
-function not_HTTP_message_filter(level, message, _module, group, id, file, line; kwargs...)
-	_module != HTTP
+function not_HTTP_message_filter(log)
+	log._module != HTTP
+end
+
+global_logger(EarlyFilteredLogger(not_HTTP_message_filter, global_logger()))
+```
+
+## Raising HTTP debug level errors to be Info level
+
+```
+using LoggingExtras
+using Logging
+using HTTP
+
+transformer_logger(global_logger()) do log
+	if log._module == HTTP && log.level=Logging.Debug
+		# Merge can be used to construct a new NamedTuple
+		# which effectively is the overwriting of fields of a NamedTuple
+		return merge(log, (; level=Logging.Info))
```

Comment:
This is cool, and I think it's basically the right idea.

The only thing I don't like is that it looses the original context that the messages from HTTP were emitted as `@debug` level.  I think the problem is that `LogLevel` has two orthogonal concepts packed into one entity:

* A categorical concept: a broad "type of thing" that the event is
* A ordinal concept: the priority (ie, the verbose -> severe axis)

I feel that transformers should be encouraged to change the priority, but not the category. I think we can retrofit this somehow into the existing system without backward incompatibilities!

Reply:
difficult

Reply:
Maybe not. I was thinking something like mapping

`LogLevel -> (category, priority)`

inside the first `handle_message(all_the_parameters)` while gathering `all_the_parameters` into a `LogRecord` or some such.

This also provides a cleaner interpretation of custom log levels which can then define their own version of this mapping rather than the current system where they define a conversion to `LogLevel` for the purposes of assessing priority.

Again, something to resolve in the stdlib rather than here.